### ISSUE TITLE
Can't open Conventient effect menu after update

### ISSUE THREAD

**Issue by Ikabodo** [DFreds/dfreds-convenient-effects#236]
**Describe the bug**
It looks like the fix on nested effects broke something in the last update. 

**To Reproduce**
Click the "Add convenient effects" button. Check error log.

**Expected behavior**
Window to choose effect should open. 

**Screenshots**

**Additional context**
```
Foundry VTT | Rendering ConvenientEffectsApp
foundry.js:747 TypeError: An error occurred while rendering ConvenientEffectsApp 185. Cannot read properties of undefined (reading 'changes')
[Detected 1 package: dfreds-convenient-effects]
    at handlebar-helpers.js:70:40
    at Array.flatMap (<anonymous>)
    at ActiveEffect5e.<anonymous> (handlebar-helpers.js:69:40)
    at ActiveEffect5e.c (handlebars.min.js:27:19853)
    at eval (eval at createFunctionContext (handlebars.min.js:29:1808), <anonymous>:21:147)
    at h (handlebars.min.js:27:17870)
    at c (handlebars.min.js:27:6899)
    at Object.<anonymous> (handlebars.min.js:27:7289)
    at Object.c (handlebars.min.js:27:19853)
    at eval (eval at createFunctionContext (handlebars.min.js:29:1808), <anonymous>:17:49)
    at h (handlebars.min.js:27:17870)
    at c (handlebars.min.js:27:6899)
    at Object.<anonymous> (handlebars.min.js:27:7289)
    at Object.c (handlebars.min.js:27:19853)
    at Object.eval [as main] (eval at createFunctionContext (handlebars.min.js:29:1808), <anonymous>:15:49)
    at c (handlebars.min.js:27:15429)
    at d (handlebars.min.js:27:15744)
    at e (handlebars.min.js:28:22748)
    at renderTemplate (foundry.js:6948:10)
    at async ConvenientEffectsApp._renderInner (foundry.js:5271:16)
    at async ConvenientEffectsApp._render (foundry.js:5114:19)
    ```

**Comment by Ikabodo** [comment]
Ehm. Looks like it's still there after a revert to 4.1.1. I'll need to investigate.

**Comment by Ikabodo** [comment]
I figured this out. It was not related to the upgrade after all.
Turned out I had created a custom nested effect, then foolishly renamed one of the nested ones. This caused the module to break when trying to load the menu since it was reffering to an effect that doesn't exist. I could fix it by editing items.db.

I think you can close this, but I would consider adding a fail safe if a player renames an effect that is part of a nested effect.

**Comment by DFreds** [comment]
Ah, that's a great failsafe to add. I'll keep this open and fix it

**Comment by DFreds** [comment]
Fixed in v4.1.3
